src/utilities/FormManager.ts

```diff
@@ -96,6 +96,7 @@ class FormManager {
   }
 
   private handleBackendResponse (response?: any) {
+    // istanbul ignore next
```

Comment:
Why did you have to add this? It's the only change in the file.

Reply:
It was an uncovered line that is not worth covering because it represents a 'This should never happen' type check / fallback error.